int8 STATUS_BORN = 0
int8 STATUS_TRACKING = 1
int8 STATUS_LOST = 2
int8 STATUS_DEAD = 3

string id
int8 status
float32 x
float32 y
float32 heading
float32 velocity
float32 sigma_x
float32 sigma_y